src/main/kotlin/com/gatecheck/gatecheck/api/RequestController.kt

```diff
@@ -0,0 +1,34 @@
+package com.gatecheck.gatecheck.api
+
+//import com.gatecheck.gatecheck.api.template.DefaultUserResponse
```

Comment:
I see lots of places with improper code conventions, understandable since you just started coding in Kotlin but I want to help you improve :P
There should be a single space between the comment `//` and the actual comment text